includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2687,7 +2687,7 @@ public function get_setup_help_text() {
 	 * @return string Connection URL.
 	 */
 	public function get_connection_url() {
-		return html_entity_decode( WC_Payments_Account::get_connect_url() );
+		return html_entity_decode( WC_Payments_Account::get_connect_url( 'WCADMIN_PAYMENT_TASK' ) );
```

Comment:
`WCADMIN_PAYMENT_TASK` has been already handled here but we missed it in #2014: 

https://github.com/Automattic/woocommerce-payments/blob/302718bc58daa13775f46e9ed1eb44796c479147/includes/class-wc-payments-account.php#L646-L646